test/include-service.thrift

```diff
@@ -0,0 +1,9 @@
+include common "./include-types.thrift"
```

Comment:
Oh you also implemented include-as? I would put a note somewhere in the documentation for this feature that this makes Thrift files incompatible with Apache Thrift. Also, since Python doesn't support this yet, probably mention the at-your-own-risk thing.